frontend/src/components/pages/eventCard.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Card } from '@mui/material';
+
+const StyledCard = styled(Card)`
+  border-radius: 6px;
+  padding: 14px;
+  padding-left: 25px;
+
+  margin-bottom: 10px;
+  text-align: left;
+`;
+
+const EventTitle = styled.p`
+  font-weight: bold;
+  font-size: 14px;
+  color: black;
+`;
+
+type EventCardProps = {
+  title: string;
+  location: string;
+};
+
+/*  for some reason background color for card would not chnage without inline css    */
```

Comment:
Hmm, that is odd. Perhaps use a Container instead of a Card? I saw that Ritvik used a container and the styling seems to work on that component. Or a button or link (see comment below).  
Whichever you choose, please remove the shadow effect on the boxes 👍